modules/light-clients/08-wasm/types/vm_test.go

```diff
@@ -242,6 +249,65 @@ func (suite *TypesTestSuite) TestWasmSudo() {
 			},
 			types.ErrWasmInvalidResponseData,
 		},
+		{
+			"failure: invalid clientstate type",
```

Comment:
Should we add similar tests in `TestWasmMigrate` (except for the one that changes the code hash)?